migrations/2_deploy_contracts.js

```diff
@@ -1,8 +1,8 @@
-// var ConvertLib = artifacts.require("./ConvertLib.sol");
-// var MetaCoin = artifacts.require("./MetaCoin.sol");
+const  ZombieAttack=artifacts.require("ZombieAttack");
+const SafeMath=artifacts.require("SafeMath");
 
-// module.exports = function(deployer) {
-//   deployer.deploy(ConvertLib);
-//   deployer.link(ConvertLib, MetaCoin);
-//   deployer.deploy(MetaCoin);
-// };
+module.exports = function(deployer) {
+  deployer.deploy(SafeMath);
```

Comment:
Are we sure now that this is the right approach? What about the updateContract routines?

Reply:
The correct deployement script is on master. This file will be ignored in the merge. What are your thought on the update contract routine?

I was of the opinion that we have a suicide function within the solidity contract and if the contract address is changed then we call suicide on the previous address